doc/nrf/libraries/networking/lwm2m_server_setup.rst

```diff
@@ -0,0 +1,95 @@
+.. _server_setup_lwm2m:
+
+Server setup
+------------
+
+The following instructions describe how to register your device to `Leshan Demo Server`_ or `Coiote Device Management server`_:
+
+1. For adding the device to the LwM2M server, complete the following steps and for adding the device to an LwM2M bootstrap server, see the procedure in :ref:`registering the device to an LwM2M bootstrap server <bootstrap_server_reg>`:
+
+   .. tabs::
+
+      .. tab:: Leshan Demo Server
+
+         1. Open the `Leshan Demo Server web UI`_.
+         #. Click :guilabel:`SECURITY` in the upper right corner in the UI.
+         #. Click :guilabel:`ADD SECURITY INFORMATION`.
+         #. Enter the following data and click :guilabel:`ADD`:
+
+            * Endpoint - urn\:imei\:*your Device IMEI*
```

Comment:
Please provide instructions, or link to instructions, on how to retrieve the IMEI.